helm/README.md

```diff
@@ -1,7 +1,115 @@
-This project is a component of the [Operator Framework](https://github.com/operator-framework), an open source toolkit to manage Kubernetes native applications, called Operators, in an effective, automated, and scalable way. Read more in the [introduction blog post](https://coreos.com/blog/introducing-operator-framework).
+# Memcached Helm Operator
 
-# Operator SDK Samples - Helm
-This directory contains samples of operators powered by Helm built using the [operator-sdk][operator_sdk]. To learn more about creating an operator that leverages Helm, check out the [Helm user guide][helm_user_guide].
+## Overview
 
-[operator_sdk]:https://github.com/operator-framework/operator-sdk
-[helm_user_guide]:https://sdk.operatorframework.io/docs/helm/quickstart/
+This Memcached operator is a simple example of the Operator SDK Helm-based operator. It is based on the [`stable/memcached` chart][stable/memcached] .
+
+## Prerequisites
+
+- [docker][docker_tool] version 17.03+
+- [kubectl][kubectl_tool] v1.14.1+
+- [operator SDK][operator_install]
+- Access to a Kubernetes v1.16.0+ cluster.
+
+## Getting Started
+
+### Clone the repository
+
+Checkout this Memcached operator repository
+
+```
+$ mkdir operator-framework
+$ cd operator-framework
+$ git clone https://github.com/operator-framework/operator-sdk-samples.git
+$ cd operator-sdk-samples/helm/memcached-operator
+```
+
+### Build the operator image
+
+Build the Memcached operator image and push it to a public registry, such as quay.io:
+
+```
+export IMAGE=quay.io/example-inc/memcached-operator:v0.0.1
+make docker-build docker-push IMG=$IMAGE
+```
+
+**NOTE** To allow the cluster pull the image the repository needs to be set as public or you must configure an image pull secret.
+
+
+### Run the operator
+
+Install the CRD and deploy the project to the cluster. Set IMG with make deploy to use the image you just pushed:
+
+```
+make install
+make deploy IMG=$IMAGE
+```
+
+### Create a sample custom resource
+
+Create a sample CR:
+
+```sh
+kubectl apply -f config/samples/cache_v1alpha1_memcached.yaml -n memcached-operator-system
+```
+
+Run the following command to verify that the installation was successful:
+
+```sh
+$ kubectl get all -n memcached-operator-system
```

Comment:
```suggestion
kubectl get all -n memcached-operator-system
```

Reply:
prefixing commands with `$` is pretty common in SDK docs since it differentiates command output from the commands themselves. If we're using `sh` blocks _without output_, removing `$` makes sense. Otherwise we should be using `$ cmd` and `console` code blocks.

Reply:
Make sense 👍